dev-utils/src/commands/test_vector_command.rs

```diff
@@ -261,6 +261,7 @@ impl VectorFactory {
         let tx = Tx::new_with(inputs.clone(), outputs.clone(), attributes);
         let tx_id = tx.id();
         let proof = TestVectorWallet::gen_proof(public_key)?.unwrap();
+        // FIXME: schnorr sign is non-deterministic -- used a fixed randomness
```

Comment:
What does this mean?

Reply:
For the test vectors, one would want to be able to byte-per-byte reproduce them (with the independent implementation, e.g. in a HW wallet) -- this isn't possible if there's some hidden randomness.
I exposed this auxiliary function where one can provide directly the random payload: https://github.com/crypto-com/rust-secp256k1-zkp/blob/upstream-catchup/src/schnorrsig.rs#L214 so the test vector generation will need to use this + also report what the random payload was

Reply:
"used a fixed randomness" -> "use a fixed randomness"

Reply:
I'll correct the comment